Description: Tentative fix for a test suite failure after the last change
 The exception fixed is the following:
  NoReverseMatch: Reverse for 'detail' with arguments '('',)' and keyword arguments '{}' not found. 1 pattern(s) tried: [u'project/volumes/(?P<volume_id>[^/]+)/$']
 .
 The test data doesn't include a volumeId attribute to volumes but they contain
 an id attribute.
 .
 Here I modify the template to rely on the the "id" attribute but maybe real
 data has a volumeId attribute and it's the test data in
 openstack_dashboard/test/test_data/nova_data.py that needs to be modified
 instead.
From: Raphael Hertzog <hertzog@debian.org>
Date: Mon, 4 Aug 2014 23:32:03 +0200

diff --git a/openstack_dashboard/dashboards/project/instances/templates/instances/_detail_overview.html b/openstack_dashboard/dashboards/project/instances/templates/instances/_detail_overview.html
index 286aef8..0968218 100644
--- a/openstack_dashboard/dashboards/project/instances/templates/instances/_detail_overview.html
+++ b/openstack_dashboard/dashboards/project/instances/templates/instances/_detail_overview.html
@@ -119,11 +119,11 @@
     {% for volume in instance.volumes %}
         <dt>{% trans "Attached To" %}</dt>
         <dd>
-            <a href="{% url 'horizon:project:volumes:volumes:detail' volume.volumeId %}">
+            <a href="{% url 'horizon:project:volumes:volumes:detail' volume.id %}">
             {% if volume.name %}
                 {{ volume.name }}
             {%  else %}
-                {{ volume.volumeId }}
+                {{ volume.id }}
             {% endif %}
             </a>
             <span> {% trans "on" %} {{ volume.device }}</span>
-- 
2.0.1

